### PR TITLE
Add information about Admins to User management page

### PR DESCRIPTION
**Pull request by radwo**

### CODE REVIEW

File: user-management-and-permissions_5c07ccc604286304a71d0252.md [hunk 1]
```diff
@@ -33,11 +35,57 @@ Owners can:
...
+card information, and see the current spending.
+
+#### Organization settings
+
+Owner can change the name of the organization. Note that this will not change
+the ID and unique URL of the organization.
+
+## Admin
+
+Admins have the same rights as the owner, and there can be multiple admins.
```
Comment (ivanaur) [inline user-management-and-permissions_5c07ccc604286304a71d0252.md]: But they can't delete an organization and request transferring ownership, right?

Comment (radwo) [reply user-management-and-permissions_5c07ccc604286304a71d0252.md]: It's about what you can do via UI. We will update this page when we introduce delete org/transferring ownership into UI.

File: user-management-and-permissions_5c07ccc604286304a71d0252.md [hunk 2]
```diff
@@ -12,13 +12,15 @@ Owner can:
 
 - See all projects;
 - Add projects, if they have admin access to the repository on GitHub;
-- Delete projects.
+- Delete projects if they have admin access to the repository on Github;
+- Manage people;
+- Manage billing;
 
-### Owner's area
+### Owner Owner's area access
```
Comment (markoa) [inline user-management-and-permissions_5c07ccc604286304a71d0252.md]: Are we seriously calling it Owner Owner? :)

Comment (radwo) [reply user-management-and-permissions_5c07ccc604286304a71d0252.md]: A little bit unfortunate ;)

**Pull request merged**